ArduPlane/quadplane.cpp

```diff
@@ -2747,22 +2757,42 @@ void QuadPlane::Log_Write_QControl_Tuning()
  */
 int8_t QuadPlane::forward_throttle_pct(void)
 {
+    // disable forward motor if disarmed, spooled down or not in a VTOL mode
+    if (!in_vtol_mode() ||
+        (SRV_Channels::get_output_channel_mask(SRV_Channel::k_throttle) &&
```

Comment:
i don't think we should check for k_throttle here

Reply:
added a boolean arg to forward_throttle_pct, as suggested